app/Http/Controllers/AdminController.php

```diff
@@ -13,6 +13,16 @@ public function dashboard(): View
     {
         $all = DB::table('customers')->get()->toArray();
         $customers = array_filter($all, fn($customer) => !$customer->is_archived);
+        $propertyToCheck = 'dates';
+        $defaultValue = [];
+
+
+        foreach ($customers as $customer) {
```

Comment:
Momentálně v DB není založená další tabulka na dates, tzn. dates property se vrací null, potřeba před vrácením inicializovat prázdným polem

Reply:
Neee delas z komara velblouda, v database/migrations upravis migraci a pridas DateTime pole date a pustis znovu migraci, pak si date vytahnes jako instanci DateTimeImmutable a muzes s tim dal pracovat

Reply:
Tak záleží, jestli chceš mít možnost customerovi přidat více dates, ale pokud ne, jasně, můžeš mít jednu column Date. Další věc, migrace se přece vytváří a zakládají na základě schématu (modelu) databáze, ne? Vytvoříš model "User", ten bude mít properties name, surname. Vytvoříš migraci, pustíš migraci. Založí se tabulka s columns name a surname na základě migrace 1. Pokud modelu přidáš property age, vytvoříš migraci a pustíš migraci, tak migrace 2 oproti první migraci založí dodatečně column age. Psát vlastní migrace ručně nedává smysl, ne? Jen mě zajímá jak to děláš ty v Laravelu

Reply:
Tak dokud to neni naplneny produkcnimi daty tak si ty migrace muzes upravovat jak chces ne?